CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-## v8.1.2 (Backport of v12.0.1)
+## v8.1.2
+
+**This release backports specific changes from v12.0.1 and v14.0.0**
```

Comment:
The CHANGELOG in master said 8.1.2 was an upcoming release, but I've checked Pypi, and you're right, it was released.

This is why CHANGELOGs should be the same across branches.